ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__automated-release-stable-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -0,0 +1,107 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+  vsphere-ci-python:
+    name: vsphere-python
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.17.0-0
+        upper: 4.18.0-0
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.17.0-0
+        upper: 4.18.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-f999
+  cron: 8 8 29 2 *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-autorelease-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-aws-ipi
+- as: azure-ipi-f999
+  cron: 8 8 29 2 *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-autorelease-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FIPS_ENABLED: "true"
```

Comment:
Looks like the job name does not indicate we are testing fips ?

Reply:
yes, we need to adjust the job name with keyword fips. and align the same changes b/w releases.

Reply:
e.g.  `azure-ipi-fips-f999`

Reply:
@LuboTerifaj

Reply:
I would say this is an issue that should be addressed in a separate jira.

Reply:
Tracked by https://issues.redhat.com/browse/OCPERT-38